packages/platform-ios/src/commands/runIOS/index.ts

```diff
@@ -128,9 +128,24 @@ async function runOnSimulator(
     throw new CLIError('Could not parse the simulator list output', error);
   }
 
-  const selectedSimulator = findMatchingSimulator(simulators, args.simulator);
+  /**
+   * If no simulator was chosen, try simulators in following order
+   * - iPhone 11
+   * - iPhone X
+   * - iPhone 8
+   */
+  let simulator: string = args.simulator;
+  if (!simulator) {
+    simulator = findMatchingSimulator(simulators, 'iPhone 11')
+      ? 'iPhone 11'
+      : findMatchingSimulator(simulators, 'iPhone X')
+      ? 'iPhone X'
+      : 'iPhone 8';
+  }
+
+  const selectedSimulator = findMatchingSimulator(simulators, simulator);
```

Comment:
this is calling `findMatchingSimulator` twice for iPhone 11 and iPhone X. Can we move this call to line 143 (iPhone 8) and get rid of `selectedSimulator` variable?